hacking.md

```diff
@@ -262,7 +263,7 @@ operator-courier --verbose verify ./deploy/olm-catalog/akka-cluster-operator
 ```
 
 Given a working Kubernetes cluster, including minikube, one can run the `scorecard` tests.
-With `operator-sdk` version v0.11.0, a config file is used to run scorecard, `.osdk-scorecard.yaml`.
+With `operator-sdk` version v0.14.1, a config file is used to run scorecard, `.osdk-scorecard.yaml`.
```

Comment:
Up in  .github/action/operator-sdk/Dockerfile, this was changed to `0.17.2`.  Should this say the same?